crates/schemalate/src/firebolt/firebolt_schema_builder.rs

```diff
@@ -166,6 +189,35 @@ pub fn build_drop_query(table: &Table) -> Result<String, Error> {
     Ok(DropTable { table }.to_string())
 }
 
+fn projection_implicit_type_to_firebolt_type(inference: &Inference) -> Option<FireboltType> {
+    let t_type = &inference.types;
+    if t_type.len() > 1 {
```

Comment:
I think we want to store values with multiple types as stringified JSONs in a Firebolt `text` field

Reply:
Unfortunately we won't be able to parse those values with EXTERNAL TABLE. It's somewhat strict in this regard so if the encountered JSON type is not a quoted string then parsing into TEXT will fail.
There's a possibility to use [COPY FROM](https://docs.firebolt.io/sql_reference/commands/data-management/copy-from.html) command instead of external tables, since it's more flexible in terms of data types it can parse, but we'd need to consider any edge cases and issues that can arise from this. This would have to be a separate change.